src/views/AssignPickerModal.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>Assign Pickers</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+ <ion-content class="ion-padding">
+   <ion-searchbar />  
+   <div class="filters">
+     <ion-row class="filter-scroll">
```

Comment:
you have already added display flex to filters, you don't need ion-row

Reply:
Done